Activities/Cryptography/UiPath.Cryptography.Activities/DecryptFile.cs

```diff
@@ -172,6 +181,31 @@ protected override void Execute(CodeActivityContext context)
                 {
                     fileName = outputFileName;
                 }
+                else
+                {
+                    fileName = Path.GetFileNameWithoutExtension(inputFilePath) + "_Decrypted" + Path.GetExtension(inputFilePath);
+                    if (string.IsNullOrEmpty(Path.GetDirectoryName(inputFilePath)))
+                    {
+                        filePath = fileName;
+                    }
+                    else
+                    {
+                        filePath = Path.GetFullPath(Path.GetDirectoryName(inputFilePath)) + "\\" + fileName;
```

Comment:
Use path composition methods, don't hardcode the separator here as it's different between OS-es.

Extract the file composition code to a helper method, use it throughout the activities.

Reply:
done